src/ralph/scan/plugins/ssh_ssg.py

```diff
@@ -100,8 +100,8 @@ def _ssh_ssg(ip_address, user, password):
 
 
 def scan_address(ip_address, **kwargs):
-    if kwargs.get('http_family') not in ('SSG', 'Unspecified'):
```

Comment:
Plugin will not run. I believe that autoscan should detect if device is an SSG and fill http_family parameter. If 'Unspecified' stays and you scan device (with http_family='Unspecified') then plugin can freeze all scanning process.